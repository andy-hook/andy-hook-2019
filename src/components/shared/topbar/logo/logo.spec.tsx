import React from "react"
import { shallowWithTheme } from "@test-utils"

import Logo from "./logo"

describe("<Logo />", () => {
  it("renders correctly", () => {
    const tree = shallowWithTheme("dark", <Logo />)
    expect(tree).toMatchSnapshot()
  })
})
