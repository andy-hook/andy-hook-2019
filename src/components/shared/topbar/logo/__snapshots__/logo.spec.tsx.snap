// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders correctly 1`] = `
<a
  className=""
  href="/"
  onClick={[Function]}
>
  <svg
    className="sc-bdVaJa cKVCxQ"
    viewBox="0 0 78 78"
  >
    <defs>
      <linearGradient
        id="LogoLinearGradient"
        x1="-35%"
        x2="86%"
        y1="20%"
        y2="73%"
      >
        <stop
          className="stop-one"
          offset="0%"
        />
        <stop
          className="stop-two"
          offset="100%"
        />
      </linearGradient>
    </defs>
    <rect
      fill="url(#LogoLinearGradient)"
      height="55"
      rx="5"
      transform="rotate(45 38.89 38.536)"
      width="55"
      x="11.391"
      y="11.036"
    />
    <path
      d="M41.126 36.264l4.377-11.338c.218-.592-.699-.98-1.208-.499L28.363 40.424c-.412.414-.038.972.585.907l5.923-.623-4.374 11.366c-.217.592.699.98 1.208.499l15.932-15.997c.412-.414.036-1-.588-.935l-5.923.623z"
      fill="#FFF"
    />
  </svg>
</a>
`;
